openerp/addons/base/security/base_security.xml

```diff
@@ -153,6 +157,37 @@
             <field name="domain_force">['|',('company_id','=',False),('company_id','child_of',[user.company_id.id])]</field>
         </record>
 
+        <!-- Security restriction for private addresses -->
+        <record id="res_partner_rule_private_employee" model="ir.rule">
+            <field name="name">res.partner.rule.private.employee</field>
+            <field name="model_id" ref="base.model_res_partner"/>
+            <field name="domain_force">
+                [('type', '!=', 'private')]
```

Comment:
@tivisse 

 The backport of this changes made in v11.0 https://github.com/Vauxoo/odoo/commit/28179bc8cb9d98a60936634e50ca733beec0393f

it's provoking an access error across accounting app and sale app
![image](https://user-images.githubusercontent.com/10233975/41362352-c857b5da-6ef6-11e8-81ac-fe296f8b92ba.png)

This rule should be changed to `[('type', 'not in', ['private'])]`

because access error is triggered doesn't matter if the type field is not set

Regards!

Reply:
I already found it and propose #25077, which has been merged and will be forward-ported soon. Meanwhile, you can change manually the record rule.

Reply:
excellent!

Thanks @pedrobaeza 

Regards!